gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Issue - Fix waiter matchers to cover object expected value type.
```

Comment:
I didn't see any changes in the lib - so we won't need this changelog.  Also - was there any diffs when you rake build all the service gems?

Reply:
Yes, no changelog is needed here.